<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentr</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Segmentr.png</normaloff>Segmentr.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTabWidget" name="Scan_Tabs">
            <property name="maximumSize">
             <size>
              <width>166</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="Scan_Tab">
             <attribute name="title">
              <string>Scan</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="0" column="0">
               <widget class="QListWidget" name="Scan_List">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Nodule_Tab">
             <attribute name="title">
              <string>Nodule</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="0">
               <widget class="QListWidget" name="Nodule_List">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Region Grow</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="0">
               <widget class="QFrame" name="frame_5">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="4">
                  <widget class="QSlider" name="RG_Slice_Slider_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSlider" name="RG_Slice_Slider">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QGraphicsView" name="RGgraphicsView2"/>
                 </item>
                 <item row="0" column="2">
                  <widget class="QGraphicsView" name="RGgraphicsView"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QFrame" name="frame_6">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="5">
                  <widget class="QPushButton" name="Edit_Button">
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Edit</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Zoom:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="RG_Zoom_Slider">
                     <property name="maximumSize">
                      <size>
                       <width>120</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="3">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Growth:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="Growth_Slider">
                     <property name="maximumSize">
                      <size>
                       <width>120</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="Seed_Button">
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Grow</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="maximumSize">
                      <size>
                       <width>36</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Slices: -</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="RG_Slice_Spin_Down">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_10">
                     <property name="maximumSize">
                      <size>
                       <width>38</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>through</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="RG_Slice_Spin_Up">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Edit</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QFrame" name="frame_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="7">
                  <widget class="QLabel" name="label_2">
                   <property name="maximumSize">
                    <size>
                     <width>52</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Brush Size:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="Mask_CheckBox">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Mask</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QRadioButton" name="RB_Draw">
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Draw</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="RB_Erase">
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Erase</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="4">
                  <widget class="QPushButton" name="Fill_Button">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Fill</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="8">
                  <widget class="QSlider" name="Brush_Slider">
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QCheckBox" name="Contours_CheckBox">
                   <property name="maximumSize">
                    <size>
                     <width>70</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Contours</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="9">
                  <widget class="QPushButton" name="PushButton_Save">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Save</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QPushButton" name="Load_Button">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Load</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QFrame" name="Editing_Frame">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="Editor_Layout">
                   <item>
                    <widget class="QGraphicsView" name="Edit_GraphicsView"/>
                   </item>
                   <item>
                    <widget class="QSlider" name="Edit_Slider">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_Base_Directory"/>
    <addaction name="Action_Save"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_Base_Directory">
   <property name="text">
    <string>Set Base Directory</string>
   </property>
  </action>
  <action name="actionSave_Segmentation">
   <property name="text">
    <string>Save Segmentation</string>
   </property>
  </action>
  <action name="actionOpen_Segmentation_2">
   <property name="text">
    <string>Open Segmentation</string>
   </property>
  </action>
  <action name="actionGenerate_All">
   <property name="text">
    <string>Generate All</string>
   </property>
  </action>
  <action name="Action_Save">
   <property name="text">
    <string>Save Segmentation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
